### PR TITLE
Change SVT-HEVC dependency path and add no-pseudo option in packing

### PR DESCRIPTION
**Pull request by starwarfan**
Change SVT-HEVC dependency path and add no-pseudo option in packing

### CODE REVIEW

File: source/agent/video/videoMixer/videoMixer_sw/binding.sw.gyp [hunk 1]
```diff
@@ -53,12 +53,6 @@
             'sources': [
                 '../../../../core/owt_base/SVTHEVCEncoder.cpp',
             ],
```
Comment (daijh) [inline source/agent/video/videoMixer/videoMixer_sw/binding.sw.gyp]: Why remove the libs link?

File: source/core/owt_base/SVTHEVCEncoder.h [hunk 2]
```diff
@@ -16,7 +16,7 @@
 #include "logger.h"
 #include "MediaFramePipeline.h"
 
-#include "EbApi.h"
+#include "svt-hevc/EbApi.h"
```
Comment (daijh) [inline source/core/owt_base/SVTHEVCEncoder.h]: why change this path?

File: source/agent/video/videoMixer/videoMixer_sw/binding.sw.gyp (hunk 1 above)
Comment (starwarfan) [reply source/agent/video/videoMixer/videoMixer_sw/binding.sw.gyp]: i installed them to build/libdeps/build as other dependencies

Comment (daijh) [reply source/agent/video/videoMixer/videoMixer_sw/binding.sw.gyp]: Suggest to keep source code include path unchanged, and only modify .gyp

File: source/core/owt_base/SVTHEVCEncoder.h (hunk 2 above)
Comment (starwarfan) [reply source/core/owt_base/SVTHEVCEncoder.h]: qiujiao need use svt-hevc installed in /usr,
this is the path generated after "make install" (PREFIX/include/svt-hevc/EbApi.h)

File: source/agent/video/videoMixer/videoMixer_sw/binding.sw.gyp (hunk 1 above)
Comment (starwarfan) [reply source/agent/video/videoMixer/videoMixer_sw/binding.sw.gyp]: link updated

**Pull request merged**